src/components/Executions/ExecutionDetails/ExecutionNodeViews.tsx

```diff
@@ -60,7 +60,10 @@ export const ExecutionNodeViews: React.FC<ExecutionNodeViewsProps> = ({
         <NodeExecutionsRequestConfigContext.Provider
             value={nodeExecutionsRequestConfig}
         >
-            <NodeExecutionsTable nodeExecutions={nodeExecutions} />
+            <NodeExecutionsTable
+                execution={execution}
+                nodeExecutions={nodeExecutions}
+            />
```

Comment:
@Pianist038801 what about to pass down `execution.closure.abortMetaData` only?
I think the other fields are not being used in the `NodeExecutionsTable`.